marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/ingestion/bulkIngester.sjs

```diff
@@ -36,7 +36,7 @@ const inputs =
       [{UNKNOWN: input}];
 inputs.forEach(record => {
   state.next = state.next + 1;
-  const uri = (state.prefix) + '/' + (work.taskId) + '/' + (state.next) + '.json';
+  const uri = (state.uriprefix) + '/' + sem.uuidString() + '.json';
```

Comment:
Just realized that we should leave out "/". That allows Ernie to have URIs of the form "(uuid).json" if he wants. That's consistent with how MLCP works as well - i.e. no "/" is added unless the user configures one.